conanfile.txt

```diff
@@ -1,22 +1,19 @@
 [requires]
-boost/1.71.0
+boost/1.83.0
 
 [generators]
 cmake_find_package
 json
 
 [options]
 boost:header_only=True
-boost:without_atomic=True
 boost:without_chrono=True
-boost:without_container=True
 boost:without_context=True
 boost:without_contract=True
 boost:without_coroutine=True
 boost:without_date_time=True
 boost:without_exception=True
 boost:without_fiber=True
-boost:without_filesystem=True
```

Comment:
this is also pretty annoying.. it wouldn't let me "build" the header-only boost without including those. IIRC filesystem is actually a compiled library. but even so, I couldn't see that conan compiles anything, so it's probably just an optional dependency of some other library

Reply:
I'd like to move to vcpkg as soon as possible after merging this PR